index.js

```diff
@@ -2,60 +2,77 @@ var dir = require("node-dir");
 var path = require("path");
 var fs = require("fs");
 
-module.exports = function(folder, cb){
+module.exports = function(folder, cb) {
 
-	dir.files(folder, function(err, files) {
-		if(err){
-			cb(err);
+	var cwd = process.cwd();
+	process.chdir(folder);
```

Comment:
it eliminates the need to do `var file = files[i].replace(folder, "");` on every filename, which in addition to being unnecessary (if you chdir), is kind of hacky, which always worries me.